src/autofill-utils.js

```diff
@@ -137,6 +138,10 @@ const setValueForSelect = (el, val) => {
     const isZeroBasedNumber = isMonth &&
         el.options[0].value === '0' && el.options.length === 12
 
+    // Ignore everything if the select element already contains the value that
+    // we're about to set. Related to https://app.asana.com/0/1177771139624306/1201923546020812/f
+    if (el.value === val) return false
```

Comment:
We should also make this check below when evaluating the `option` text content. Sometimes selects have different values and text content and we can match one but not the other (i.e. country codes vs country names, month names vs month number, etc.).

Reply:
@GioSensation please see the updated changeset - I think we can do an even simpler approach and just never set an option if already selected - I tested on paypal and this still works well

Reply:
Yep, it's nicer now.